dev/src/dev/migration.clj

```diff
@@ -0,0 +1,62 @@
+(ns dev.migration
+  (:require
+   [metabase.db :as mdb]
+   [metabase.db.liquibase :as liquibase]
+   [toucan2.core :as t2])
+  (:import
+   (liquibase Liquibase)))
+
+(set! *warn-on-reflection* true)
+
+(defn- latest-migration
+  []
+  ((juxt :id :comments)
+   (t2/query-one {:select [:id :comments]
+                  :from   [:databasechangelog]
+                  :order-by [[:orderexecuted :desc]]
+                  :limit 1})))
+(defn migrate!
+  "Run migrations for the Metabase application database. Possible directions are `:up` (default), `:force`, `:down`, and
+  `:release-locks`. When migrating `:down` pass along a version to migrate to (44+)."
+  ([]
+   (migrate! :up))
+  ;; do we really use this in dev?
+  ([direction & [version]]
+   (mdb/migrate! (mdb/db-type) (mdb/data-source)
+                 direction version)
+   (println "Latest migration:" (latest-migration))))
+
+(defn- rollback-n-migration!
+  [n]
+  (with-open [conn (.getConnection (mdb/data-source))]
+    (liquibase/with-liquibase [^Liquibase liquibase conn]
+      (liquibase/with-scope-locked liquibase
+        (.rollback liquibase n "")))))
+
+(defn- id->n-to-rollback
```

Comment:
Perhaps it's just me, but I find this name a bit awkward. How about something like `migrations-since`?